pudb/lowlevel.py

```diff
@@ -28,6 +28,39 @@
 
 from pudb.py3compat import PY3, text_type
 
+import logging
+
+
+class SafeStreamHandler(logging.StreamHandler):
+    """
+    Logging stream handler that clears up the UI when it's done, if needed.
+    """
+    def emit(self, record):
+        from pudb import _have_debugger, _get_debugger
+        super(SafeStreamHandler, self).emit(record)
+        if _have_debugger():
+            dbg = _get_debugger()
+            dbg.ui.screen.clear()
+
+
+ui_formatter = logging.Formatter(
+    fmt='\n*** Pudb UI Exception Encountered: %(message)s ***\n'
+)
+ui_handler = SafeStreamHandler()
+ui_handler.setFormatter(ui_formatter)
+ui_log = logging.getLogger('ui')
+ui_log.setLevel(logging.CRITICAL)
```

Comment:
Given fd083e4, does it still make sense to swallow errors by default? (And, similarly, does the `--log-errors` option still make sense?)

Reply:
I see your point! I've removed the option and set the errors to always be shown